src/components/Select/SharedComponents/ButtonSelectWrapper/ButtonSelectWrapper.tsx

```diff
@@ -66,7 +68,7 @@ export const ButtonSelectWrapper = (props: TSelectWrapperProps): ReactElement =>
         isOpen={isOpen}
         buttonText={placeHolder || ''}
         selectedItemsLabels={selectedItemsLabels}
-        onClick={openDropdown}
+        onClick={disabled ? noop : openDropdown}
```

Comment:
@LilitArz jan !disabled &&  openDropdown will it work? i do not know if typescript will allow i think much shorter this way